src/visGeometry/GeometryStore.ts

```diff
@@ -221,15 +221,15 @@ class GeometryStore {
             // TODO:
             // Can we confirm that the rcsb.org servers have every id as a cif file?
             // If so, then we don't need to do this second try and we can always use .cif.
-            actualUrl = `https://files.rcsb.org/download/${pdbID}-assembly1.cif`;
+            actualUrl = `https://files.rcsb.org/download${pdbID}-assembly1.cif`;
```

Comment:
See what you think of this idea: moving both of the changed lines into a single function something like getPdbUrlFromSanitizedPdbId(pdbid, is_cif_or_pdb1) and then in that function explain in a comment why a pdb id would have a leading slash here. (...because some other function already prepended it...)